pvm.py

```diff
@@ -35,116 +43,230 @@ def _init_logger():
+				else:
+					_logger.info("%s command failed.", command)
+			elif command=="set_position":
+				p1, p2 = parse_value(value)
+				media1.set_position(float(p1))
+				media2.set_position(float(p2))
 				_logger.info("%s command success.", command)
-			else:
-				_logger.info("%s command failed.", command)
-		elif command=="stop":
-			if media.can_quit():
-				media.stop()
-				IS_FILE_SET = False
-				_logger.info("%s command success and file has been unset.", command)
-			else:
-				_logger.info("%s command failed.", command)
-		elif command=="set_position":
-			media.set_position(float(value))
-			_logger.info("%s command success.", command)
-		elif command=="set_rate":
-			fps = str(30 * float(value))
-			media = OMXPlayer(PREFIX_PATH + VIDEO_PATH, dbus_name='org.mpris.MediaPlayer2.omxplayer', args=['--loop','--force-fps', fps])
-			if media != "":
-				media.quit()
-				print("media quit")
-			original_fps = 30
-			video_info = subprocess.Popen(["omxplayer", "-i", PEFIX_PATH + VIDEO_PATH], stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
-			print("The command", ["omxplayer", "-i", PEFIX_PATH + VIDEO_PATH])
-			# info = video_info.stdout.decode().split(", ")
-			out, err = video_info.communicate()
-			out = out.decode(encoding='utf-8')
-			splist = out.split(", ")
-			for s in splist:
-				if 'fps' in s:
-					print("current ele:", s)
-					fps_info = s.split(' ')
-					original_fps = float(fps_info[0])
-			fps = str(original_fps * float(value))
-			print("computed fps", fps)
-			media = OMXPlayer(PREFIX_PATH + VIDEO_PATH, dbus_name='org.mpris.MediaPlayer2.omxplayer1', args=['--loop','--force-fps', fps])
-			media.pause()
-			_logger.info("%s command success.", command)
-		elif command=="pause":
-			if media.can_pause():
-				media.pause()
+			elif command=="set_rate":
+				v1, v2 = parse_value(value)
+				fps1 = get_rate(VIDEO_PATH_ONE, v1)
+				fps2 = get_rate(VIDEO_PATH_ONE, v2)
+				media1 = OMXPlayer(VIDEO_PATH_ONE, dbus_name='org.mpris.MediaPlayer2.omxplayer1', args=['--loop', '--display', '2', '--force-fps', fps1])
+				media1.pause()
+				media2 = OMXPlayer(VIDEO_PATH_TWO, dbus_name='org.mpris.MediaPlayer2.omxplayer2', args=['--loop', '--display', '7', '--force-fps', fps2])
+				media2.pause()
 				_logger.info("%s command success.", command)
+			elif command=="pause":
+				if canPause:
+						media1.pause()
+						media2.pause()
+						pos1 = media1.position()
+						pos2 = media2.position()
+						_logger.info("media 1 position: %s", pos1)
+						_logger.info("media 2 position: %s", pos2)
+						if pos1 > pos2:
+							media1.set_position(pos2)
+							_logger.info("media 1 set position: %s", pos2)
+						elif pos1 < pos2:
+							media2.set_position(pos1)
+							_logger.info("media 2 set position: %s", pos1)
+						canPause = False
+						canStart = True
+						_logger.info("%s command success.", command)
 			else:
-				_logger.info("%s command failed.", command)
+				_logger.info("%s unknown.", command)
```

Comment:
what about the whole strategy of using the `=` symbol as a way to bypass change as discussed in #49 ?...